back-end/src/main/java/kr/co/ssalon/domain/service/MeetingService.java

```diff
@@ -53,4 +55,17 @@ public Page<Meeting> getMoims(MeetingSearchCondition meetingSearchCondition, Pag
         Page<Meeting> meetings = meetingRepository.searchMoims(meetingSearchCondition, pageable);
         return meetings;
     }
+
+    public Meeting findMeeting(Long id) throws BadRequestException {
+        Optional<Meeting> findMeeting = meetingRepository.findById(id);
+        Meeting meeting = validaitonMeeting(findMeeting);
+        return meeting;
+    }
+
+    private Meeting validaitonMeeting(Optional<Meeting> meeting) throws BadRequestException {
```

Comment:
오타났네요. `MemberService.validaitonMeeting`에도 오타가 같이 났었군요. 같이 수정해주시면 감사하겠습니다!
```suggestion
    private Meeting validationMeeting(Optional<Meeting> meeting) throws BadRequestException {
```

Reply:
헉 수정하겠습니다